documentation/modules/new-migrating-virtual-machines-cli.adoc

```diff
@@ -202,7 +202,7 @@ stringData:
 EOF
 ----
 <1> The `ownerReferences` section is optional.
-<2> Specify the {virt} user.
+<2> Instead of entering a user, create a service account and a token, as described in xref:adding-source-provider_cnv2[Adding an OpenShift Virtualization destination provider].
```

Comment:
so why not to drop the `user` from here and add `token`?

Reply:
I made the change, but take a look at where the link leads to: https://file.emea.redhat.com/rhoch/ova_ocp_network_storage_map/html-single/#adding-destination-providers_mtv

The link works fine, but id=s the information we want them to see ?

Reply:
the link doesn't take me to the right section, I believe it is `4.4.2. Adding destination providers`, right?
if so, then I think we can just duplicate the relevant part there:
```
Service account bearer token: Token for a service account with cluster-admin privileges

If both URL and Service account bearer token are left blank, the local OpenShift cluster is used.
```

Reply:
https://file.emea.redhat.com/rhoch/ova_ocp_network_storage_map/html-single/#adding-destination-providers_mtv

Reply:
This should do it.